quinn/src/recv_stream.rs

```diff
@@ -94,14 +94,17 @@ impl RecvStream {
         .await
     }
 
-    /// Attempts to read from the stream into buf.
+    /// Attempt to read from the stream into the provided buffer
```

Comment:
FWIW this goes against the Rust style guide, which I think is nice to follow generally: https://rust-lang.github.io/rfcs/1574-more-api-documentation-conventions.html#appendix-a-full-conventions-text

> The summary line should be written in third person singular present indicative form. Basically, this means write ‘Returns’ instead of ‘Return’.

Reply:
This would also be more consistent with the following paragraph 👍